src/main/java/org/sciborgs1155/lib/InputStream.java

```diff
@@ -0,0 +1,124 @@
+package org.sciborgs1155.lib;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.filter.SlewRateLimiter;
+import java.util.function.DoubleSupplier;
+import java.util.function.DoubleUnaryOperator;
+
+/**
+ * A functional interface to aid in modifying double suppliers, such as from a joystick.
+ *
+ * <p>Inspired by 694's StuyLib.
+ */
+@FunctionalInterface
+public interface InputStream extends DoubleSupplier {
```

Comment:
Would you be horribly opposed to calling this DoubleInputStream? (or using generics, but then you'd have to sacrifice your scaling -- which honestly would be totally fine imo just use map/transform)

Reply:
i would prefer not to...